New Elf-A Mutations

zoshkalin opened the pull request:
#### Summary
```SUMMARY: Content "Adds new mutations to Elf-A category"```

#### Purpose of change
Just some new mutations to help flesh Elf-A out a little more

#### Describe the solution
Changed the Elf-A recipe slightly.
ElfA is closer to plant and human than slime and bird.
2x plant
1x slime
1x bird
2x purifier

Quick as an extra pre-threshold mutation

New mutations:
Gruff Voice is the prerequisite to Pleasant Voice. It gives small benefits to intimidation, but an equal decrease in persuasion and lying. It can also evolve into Growling Voice and Hissing Voice.

Pleasant Voice is an upgrade to Gruff Voice, which gives a bonus to persuasion and lying, but decreases intimidation.

Post-threshold mutations:
Melodic Voice - The upgrade to Pleasant Voice

Animal Harmony - The upgrade to Animal Kinship. All creatures with the MF_ANIMAL flag ignore the player, regardless of initial aggressiveness

Glorious - The old Glorious is renamed to Extremely Beautiful

#### Testing
Tested Glorious and Melodic Voice by chatting with NPCs

Tested Animal Harmony by spawning 10 sewer gators (aggressiveness 30) and they all ignored me.

Fris0uman commented:
Maybe you could use the `ignored_by` entry instead of hardcoding one more mutation?

Zireael07 commented:
Gruff Voice doesn't really fit elves. The others are great!

SantosValourion commented:
"ElfA is closer to plant and human than slime and bird.
2x plant
1x slime
1x bird
2x purifier"
Did you mean to switch bird with plant?

SantosValourion commented:
If i can Make a suggestion, add a negative trait that acts like kind of like the nomad trait, except that it gives negative morale penalty for not being in woods or swamps and a positive for being in woods or swamps. Also the Nomad trait itself might also be good to add.

zoshkalin commented:
So I have made a couple changes, and a few discoveries as well

I removed Gruff Voice from Elf-A

I also added another mutation_type BEAUTY for the Ugly and Pretty mutation lines.

Everything should be up to date with the new style changes as well.

I also fixed an oversight with the Animal Kinship mutation, which was adding morale to animals that may be hostile to the player. 

There was a really strange bit of code that was adding the hit_by_player effect to friendly monsters. 
It was also adding it to sewer gators when I set their attitude to MATT_IGNORE, which was turning them hostile.
I don't know what this code was meant to be for, because it is very old and doesn't seem to have a use.

All animals are neutral to the player when they have Animal Harmony, however monsters with the ATTACKMON flag still attack by bumping into the player. I'm not sure if I should do anything about this though.

Hitting the animals changes them back to their ordinary behaviour until the hit_by_player effect runs out

@Fris0uman I mainly added Animal Harmony to finish the Animal Empathy line. As for the anger_relations, it lowers the anger permanently, which will probably prevent hostile animals from becoming hostile again after getting hit.

Night-Pryanik commented:
@zoshkalin is this PR ready?

ZhilkinSerg commented on data/json/mutations/mutations.json:
```diff
@@ -6811,4 +6876,4 @@
     "dodge_modifier": 2
   }
-]
+]
```
```suggestion
]

```

ZhilkinSerg commented:
Closing to clean up PR queue. Feel free to resolve conflicts and reopen.
